packages/cascara/src/hooks/useMediaQuery.js

```diff
@@ -3,22 +3,33 @@
 import { useEffect, useState } from 'react';
 
 const useMediaQuery = (query, ifTrueFunc, ifFalseFunc) => {
-  const mediaQuery = window.matchMedia(query);
+  const isDOM = Boolean(
+    typeof window !== 'undefined' &&
+      window.document &&
+      window.document.createElement
+  );
```

Comment:
Could we find the other place we were using this isDOM function and place it in a location where we can reuse it? I know there are going to be other scenarios where we need it moving forward. We might as well make sure we are reducing boilerplate now.

If any of those other places exist inside of the Docs project, we can disregard them for now. But anywhere we are using this in Cascara, we should just have one helper function.

Reply:
There was only one copy of the function in Boundaries. Function moved to /lib folder and both components updated.